### PR TITLE
adding guide links

### PR DESCRIPTION
**Pull request by nsdere**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #126

Test URLs:
- Before: https://main--bitdefender--hlxsites.hlx.page/solutions/internet-security
- After: https://guide-links--bitdefender--hlxsites.hlx.page/solutions/internet-security

### CODE REVIEW

**Review by lansami — APPROVED** [review]

File: styles/styles.css [hunk 1]
```diff
@@ -405,6 +405,27 @@ a.button:any-link, button {
...
+  color: var(--background-blue);
+}
+
+a.info-button.modal::before {
+  content: url('/icons/user-guide.svg');
+  display: flex;
+}
+
+p.info-button-container {
+  font-weight: 500;
```
Comment (basecode) [inline styles/styles.css:418]: please use variable for `font-weight`.

File: scripts/lib-franklin.js [hunk 2]
```diff
@@ -615,6 +615,13 @@ export function decorateButtons(element) {
           a.nextSibling.remove();
           return;
         }
+        if (up.childNodes.length === 1 && up.tagName === 'P' && up.innerText.startsWith('?')) {
```
Comment (basecode) [inline scripts/lib-franklin.js:618]: could you add a comment like `// Example: <p><a href="example.com">? Text</a></p>` please?

File: styles/styles.css [hunk 3]
```diff
@@ -405,6 +405,27 @@ a.button:any-link, button {
...
+  font-size: 1pc;
```
Comment (basecode) [inline styles/styles.css:419]: please use variable for `font-size`.

**Review by basecode — COMMENTED** [review]
looks good! minor requests.

**Pull request merged**